setup-env.sh

```diff
@@ -6,30 +6,32 @@
 # configure env
 ######################
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
 NODE_ENV=development
-echo export NODE_ENV=$NODE_ENV >> .env
+echo export NODE_ENV=$NODE_ENV > ${DIR}/.env
```

Comment:
> or >>

Reply:
rest lgtm

Reply:
Since it's the first command, I think we want to replace `.env` instead of append to it so that if the command is run multiple times, it doesn't continuously add to the file.

Reply:
So `>` was intentional